.rubocop.yml

```diff
@@ -1,34 +1,24 @@
-require:
-  - rubocop-performance
-  - rubocop-minitest
+inherit_from: .rubocop_todo.yml
 
-AllCops:
-  NewCops: enable
+inherit_gem:
+  theforeman-rubocop:
+    - strictest.yml
 
-Layout/EmptyLineAfterGuardClause:
-  Enabled: false
+AllCops:
+  TargetRubyVersion: 2.7
 
 Layout/LineLength:
   Enabled: false
 
 Metrics:
   Enabled: false
 
-Style/FormatStringToken:
```

Comment:
Same question as in https://github.com/theforeman/hammer_cli_foreman_bootdisk/pull/18/files. Can we remove all other exceptions and have the config file the same as in the https://github.com/theforeman/hammer-cli-foreman-google/blob/master/.rubocop.yml?

Reply:
@stejskalleos here we already have rubocop todo file so added some cops in there, because right now our focus is more on adding theforeman-rubocop gem and less focus on solving all the offenses because of the new style rules. but i tried to solve some.